packages/angular/src/schematics/init/init.ts

```diff
@@ -153,8 +153,7 @@ export function setDefaults(options: Schema): Rule {
 function addPostinstall(): Rule {
   return updateJsonInTree('package.json', (json, context) => {
     json.scripts = json.scripts || {};
-    const command =
-      'ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points';
```

Comment:
Please make sure to check how this changes how long this takes to execute on repos with many dependencies.

We may want to only remove the `--create-ivy-entry-points`. I passed the other flags to try and keep the execution time down since this happens after every install.

Reply:
yep, I'm exactly currently testing those things. I just have to find out why Jest tests still don't use Ivy when I leave the `--properties ...` in.

Reply:
So this is interesting:

- `ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points`: Jest & Storybook use ViewEngine
- `ngcc --properties es2015 browser module main --first-only`: Jest runs in ViewEngine, Storybook runs in Ivy
- `ngcc --properties es2015 browser module main`: Jest and Storybook run in Ivy mode

🤔

Reply:
If you pass `--first-only`, it won't generate `umd` bundles which are required for Jest to work with Ivy